app/Model/Cert.php

```diff
@@ -491,4 +491,39 @@ public function beforeSave($options = array())
     return true;
   }
 
+  /**
+   * afterSave
+   *
+   * @param  boolean true if a new record was created (rather than update)
+   * @param  array, the same passed into Model::save()
+   * @return none
+   */
+  function afterSave($created, $options = Array()) {
+    $this->manualProvisionCert();
+  }
+
+  /**
+   * afterDelete
+   *
+   * @return none
+   */
+  function afterDelete() {
+    $this->manualProvisionCert();
+  }
+  
+  /**
+   * manualProvisionCert
+   *
+   * @return none
+   */
+  function manualProvisionCert() {
+    $sModel = 'CoPerson';
+    $sAction = ProvisioningActionEnum::CoPersonUpdated;
```

Comment:
What will happen during the enrollment of a user using IGTF? Will CoPersonUpdated do the work we need?